third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/call_transfer_borrows.exp

```diff
@@ -1,19 +1,5 @@
 
 Diagnostics:
-error: cannot move local `x` which is still borrowed
-   ┌─ tests/reference-safety/v1-tests/call_transfer_borrows.move:17:9
-   │
-13 │         let x_ref = &x;
-   │                     -- previous local borrow
-14 │         let y_ref = &mut y;
-15 │         let r = take_imm_mut_give_mut(x_ref, y_ref);
-   │                 ----------------------------------- used by mutable call result
-16 │         *x_ref;
-17 │         move x; // error in v2 (bug in v1)?
```

Comment:
It looks a bit confusing that we lose this error (but looks like it is indeed safe). Should we change the comment here? What is the v1 behavior for this test case?

Reply:
V1 does not have this error

Reply:
Removed the comment in the test case